shared/.babelrc

```diff
@@ -15,26 +15,6 @@
     ]
   ],
   "env": {
-    // Used to build electron app
```

Comment:
The electron webpack build ignores babelrc files. The way babel-loader works is it'll inherit parent babelrc files so what we want to do w/ RN and our scripts 'infects' the build output